lib/response.js

```diff
@@ -368,6 +371,10 @@ res.sendFile = function sendFile(path, options, callback) {
     done = options;
     opts = {};
   }
+  
+  if (app.settings.etag == false) {
```

Comment:
You should use the setting functions to access settings, not direct access to the object. Direct object access was deprecated in 3.0.